tests/browser/index.html

```diff
@@ -1,6 +1,7 @@
 <html>
   <head>
-    <script type="text/javascript" src="../../_bundles/trustlines-network.js"></script>
+    <script src="https://cdn.ethers.io/scripts/ethers-v4.min.js" charset="utf-8" type="text/javascript"></script>
```

Comment:
I dont think it makes sense to have ethers external, if it is needed to work. 
Is this the same bundle we use in the app?

Reply:
So in general we create 3 different formats which support different approaches to import our library.
```javascript
// ./lib-esm -> ES6 
import { TLNetwork } from 'trustlines-clientlib'

// lib -> UMD format
const TLNetwork = require('webpack-numbers').TLNetwork
 
// _bundles -> loading lib via script tag
<script type="text/javascript" src="../../_bundles/trustlines-clientlib.js"></script>
```
The first 2 formats are created via `tsc` whereas the last one is the only one bundled with `webpack`. And the app uses `lib-esm`.

Reply:
https://github.com/trustlines-network/clientlib/issues/192